ui/components/multichain/notification-detail-network-fee/notification-detail-network-fee.tsx

```diff
@@ -145,11 +233,26 @@ export const NotificationDetailNetworkFee: FC<
           justifyContent={JustifyContent.flexStart}
           width={BlockSize.Full}
         >
-          <FeeDetail label="Gas limit (units)" value={gasLimit} />
-          <FeeDetail label="Gas used (units)" value={gasUsed} />
-          <FeeDetail label="Base fee (GWEI)" value={baseFee} />
-          <FeeDetail label="Priority fee (GWEI)" value={priorityFee} />
-          <FeeDetail label="Max fee per gas" value={maxFee} />
+          <FeeDetail
+            label="Gas limit (units)"
```

Comment:
I think this needs to be localized, correct?

Reply:
You are right. Fixed!